File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.py
num1, num2 = map(int,input().split())
a = []
for i in range (1, num1+1):
    a.append(int(i))
for i in range(0, num2):
    num3 ,num4 = map(int, input().split())
    temp = a[num3-1]
    a[num3-1] = a[num4-1]
    a[num4-1] = temp
for a in a:
    print(a, end=' ')

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();
        int[] array = new int[num1];

        for(int i=0; i<array.length;i++){
            array[i]=i+1;
        }
        for(int t=0; t<num2; t++){
            int i = scanner.nextInt();
            int j = scanner.nextInt();
            int k ;
            k = array[i-1];
            array[i-1]=array[j-1];
            array[j-1] = k;
        }
        for(int a: array){
            System.out.print(a + " ");
        }
    }
}